docs/runbooks/search-functionality/resident-search.md

```diff
@@ -0,0 +1,96 @@
+---
+id: resident-search
+title: Resident Search Configuration
+slug: /runbooks/search-functionality/resident-search
+---
+
+## Technical Notes
+
+### Postgres
+`pg_tgrm extension for Postgres`
+
+- The person search uses fuzzy name matching that uses the postgres trigram extension in queries. 
+- The *pg_tgrm* extension adds required functionality and also creates a new index to improve performance.
+- If the extension is not installed in your (table/database?) you will get the following error:
```

Comment:
Suggestion: If the extension is not installed in your database you will get the following error when you try to run queries that make use of trigrams:

Reply:
Sounds good thanks